frontend/components/MediaControls.cpp

```diff
@@ -196,9 +197,12 @@ void MediaControls::StopMediaTimer()
 void MediaControls::SetPlayingState()
 {
 	ui->slider->setEnabled(true);
-	ui->playPauseButton->setProperty("class", "icon-media-pause");
-	ui->playPauseButton->style()->unpolish(ui->playPauseButton);
-	ui->playPauseButton->style()->polish(ui->playPauseButton);
+
+	addClass(ui->playPauseButton, "icon-media-pause");
+	removeClass(ui->playPauseButton, "icon-media-play");
+	removeClass(ui->playPauseButton, "icon-media-restart");
```

Comment:
Would `setClasses(ui->playPauseButton, "icon-media-pause")` be more 1:1 here (and in the other `Set*State` functions) instead of explicitly doing `add`, `remove`, `remove`?